OpenRA.Mods.Common/Traits/Upgrades/UpgradeManager.cs

```diff
@@ -19,7 +19,7 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Attach this to a unit to enable dynamic upgrades by warheads, experience, crates, support powers, etc.")]
-	public class UpgradeManagerInfo : ITraitInfo, Requires<IUpgradableInfo>
+	public class UpgradeManagerInfo : ITraitInfo, InitializeAfter<IUpgradableInfo>
```

Comment:
I was planning on changing some traits to use `INotifyCreated` in a separate PR, but I guess I  can do this one here.